### ISSUE TITLE
Can you make presets into dimensions?

### ISSUE THREAD

**Issue by Abo233** [McJty/LostCities#226]

**Comment by Abo233** [comment]
I want to play  multiple presets  in one archive

**Comment by McJty** [comment]
Yes you can. Check the general.cfg. There you can define multiple dimensions each with their own profile

**Comment by thecrescentkid** [comment]
How,
can you make a tutorial video.

**Comment by McJty** [comment]
That doesn't really require a video. Did you open the config file? There you can see this config entry. Just extend/edit it like you want:

    # List of additional Lost City dimensions. Format '<id>:<profile>' [default: ]
    S:additionalDimensions <
        112:cavern
        113:floating
     >

**Comment by Abo233** [comment]
Thanks you！

**Comment by Abo233** [comment]
I'm sorry, so late back to you
